server/src/controllers/projects.controllers.ts

```diff
@@ -137,6 +140,60 @@ class ProjectsController {
 		}
 	};
 
+	updateProjectUsersList = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+		const projectsRepository = getCustomRepository(ProjectsRepository);
+		const { projectId, usersId } = req.body;
+
+		const prevProject = await projectsRepository.getOne(projectId);
+
+		if (prevProject === undefined) {
+			next(new ErrorResponse(HttpStatusCode.NOT_FOUND, projectsErrorMessages.PROJECT_NOT_FOUND));
+			return;
+		}
+		const { id: prevProjectLeadId } = prevProject.lead;
+		const { users: prevProjectUsers } = prevProject;
+
+		const isLeadInUsers = prevProjectUsers.find((user) => user.id === prevProjectLeadId);
+		const isLead = usersId === prevProjectLeadId;
+
+		if (!isLeadInUsers || isLead) {
+			next(new ErrorResponse(HttpStatusCode.UNPROCESSABLE_ENTITY, projectsErrorMessages.INVALID_DATA));
+			return;
+		}
+
+		const { id: userId } = req.user;
+
+		const isForbiddenResult = this.isForbidden(userId, prevProjectLeadId, next);
+
+		if (isForbiddenResult) {
```

Comment:
better to send with message

Reply:
The message is sent in the isForbidden function